### PR TITLE
[FR] Add `Ranking_Criteria\osu!`

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal to add the French translation of the wiki page `Ranking_Criteria\osu!`.

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+- **Évitez le [slider velocity](/wiki/Hit_object/Slider_velocity) au-dessus de 1,3.**
+- **La densité des notes doit consister principalement en des rythmes 1/1, 2/1 ou plus lents.**
+- **Évitez d'utiliser des sliders plus courts que 1/2 d'un temps.** Ceux-ci sont trop rapides pour être compris par les nouveaux joueurs.
+- **Évitez l'[overlapping](/wiki/Mapping_techniques/Overlap) des cercles, des slider heads et des slider tails.** Cela peut être perturbant pour les nouveaux joueurs. Les overlapping 1/1 dus à au [distance snapping](/wiki/Client/Beatmap_editor/Distance_snap) doivent être évités avec un espacement plus important ou du [slider velocity](/wiki/Hit_object/Slider_velocity).
+- **Évitez les sliders 1/1 avec des inversions multiples.** Les nouveaux joueurs sont incapables de lire les reverses supplémentaires car ils sont visibles pendant un temps très court.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!/fr.md:77]:
```suggestion
- **Évitez les sliders 1/1 avec plusieurs reverses.** Les nouveaux joueurs sont incapables de lire les reverses supplémentaires car ils sont visibles pendant un temps très court.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+- **Évitez les sections où il n'y a que des sliders.** Viser et suivre un grand nombre de sliders à la suite peut être fatigant pour les nouveaux joueurs. Dans ce cas, il convient d'utiliser des cercles et des moments de repos sans objets à cliquer ou à suivre.
+- **Il est déconseillé de manipuler fréquemment le [slider velocity](/wiki/Hit_object/Slider_velocity).** Le slider velocity ne doit être modifiée que pour les sections de musique avec des rythmes différents et ne doit pas être radicalement variable.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!/fr.md:79]:
```suggestion
- **Il est déconseillé de manipuler fréquemment le [slider velocity](/wiki/Hit_object/Slider_velocity).** Le slider velocity ne doit être modifié que pour les sections de musique avec des rythmes différents et ne doit pas être radicalement variable.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+- **Utilisez des formes de sliders simples et facilement compréhensibles.** Les nouveaux joueurs peuvent ne pas comprendre comment suivre des formes plus complexes.
+- **Il doit y avoir au moins 4 temps entre l'extrémité d'un spinner et l'objet suivant.** Ceci afin de garantir un temps suffisant pour cliquer sur un objet qui suit un spinner.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!/fr.md:81]:
```suggestion
- **Il doit y avoir au moins 4 temps entre la fin d'un spinner et l'objet suivant.** Cela permet de garantir un temps suffisant pour cliquer sur un objet qui suit un spinner.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+- **Évitez les spinners de moins de 4 temps.** Les joueurs ont besoin de temps pour reconnaître qu'ils doivent commencer à tourner.
+- **Évitez de recouvrir entièrement les flèches des sliders avec des objets dans les 4 temps où la flèche est atteinte.** Cela donne aux joueurs suffisamment de temps pour reconnaître la flèche.
+- **Évitez l'[overlapping](/wiki/Mapping_techniques/Overlap) complet des sliders bodies à moins de 4 temps l'un de l'autre.** En procédant ainsi, les sliders peuvent être mal considérés comme un cercle en raison de l'obscurcissement du slider body.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!/fr.md:84]:
```suggestion
- **Évitez l'[overlapping](/wiki/Mapping_techniques/Overlap) complet des slider bodies à moins de 4 temps l'un de l'autre.** En procédant ainsi, les sliders peuvent être mal considérés comme un cercle en raison de l'obscurcissement du slider body.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+
+#### Règles
+
+- **Les objets séparés de 1 temps ou moins ne doivent pas se [chevaucher](/wiki/Mapping_techniques/Overlap) complètement.**
+- **N'utilisez pas de [flèches d'inversion de slider qui ne suivent pas le chemin du slider](/wiki/Ranking_Criteria/osu!/img/Unintuitive_slider_reverse_arrow.png).** Ceux-ci déforment la direction d'un slider.
+- **Chaque slider doit avoir une trajectoire claire et visible à suivre du début à la fin.** Les sliders qui se chevauchent sans bordures de slider claires et les sliders dont les sections individuelles sont illisibles ne peuvent pas être utilisés. La position finale d'un slider doit être claire en supposant que le joueur possède un skin qui rend les cercles de fin de slider totalement transparents.
+
+#### Directives
+
+- **Le [time-distance equality](/wiki/Mapping_techniques/Time-distance_equality) doit être utilisée.** Les variations sont acceptables si elles sont clairement différentes de l'espacement utilisé pour les différents rythmes. Lorsque les objets sont espacés de plus d'un temps et que l'espacement est relativement important sur le terrain de jeu, le time-distance equality n'a pas besoin d'être aussi précis.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!/fr.md:104]:
```suggestion
- **Le [time-distance equality](/wiki/Mapping_techniques/Time-distance_equality) doit être utilisé.** Les variations sont acceptables si elles sont clairement différentes de l'espacement utilisé pour les différents rythmes. Lorsque les objets sont espacés de plus d'un temps et que l'espacement est relativement important sur le playfield, le time-distance equality n'a pas besoin d'être aussi précis.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+- **Lorsque [distance snap](/wiki/Client/Beatmap_editor/Distance_snap) est utilisé, essayez de le maintenir entre 0,8x et 1,3x.** Des valeurs trop élevées ou trop faibles peuvent rendre la différence entre le [slider velocity](/wiki/Hit_object/Slider_velocity) et l'espacement contre-intuitif. Un distance snap plus élevée peut être utilisée pour éviter l'[overlapping](/wiki/Mapping_techniques/Overlap), mais le slider velocity devrait être augmenté si vous êtes retrouvez à le faire souvent.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!/fr.md:105]:
```suggestion
- **Lorsque [distance snap](/wiki/Client/Beatmap_editor/Distance_snap) est utilisé, essayez de le maintenir entre 0,8x et 1,3x.** Des valeurs trop élevées ou trop faibles peuvent rendre la différence entre le [slider velocity](/wiki/Hit_object/Slider_velocity) et l'espacement contre-intuitif. Un distance snap plus élevé peut être utilisée pour éviter l'[overlapping](/wiki/Mapping_techniques/Overlap), mais le slider velocity devrait être augmenté si vous êtes retrouvez à le faire souvent.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 7]
```diff
@@ -0,0 +1,201 @@
...
+- **La densité des notes doit consister principalement en 1/1, occasionnellement 1/2, ou des rythmes plus lents.**
+- **Évitez l'[overlapping](/wiki/Mapping_techniques/Overlap) des cercles, des slider heads et des slider tails.** Ceux-ci peuvent être perturbants pour les nouveaux joueurs. Les overlaps 1/2 dus au [distance snapping](/wiki/Client/Beatmap_editor/Distance_snap) sont une exception.
+- **Évitez les longues chaînes d'objets avec des écarts de 1/2.** Un trop grand nombre d'objets consécutifs [chauvauché](/wiki/Mapping_techniques/Overlap) peut être source de confusion pour les nouveaux joueurs.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!/fr.md:108]:
```suggestion
- **Évitez les longues chaînes d'objets avec des écarts de 1/2.** Un trop grand nombre d'objets consécutifs [chevauchés](/wiki/Mapping_techniques/Overlap) peut être source de confusion pour les nouveaux joueurs.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 8]
```diff
@@ -0,0 +1,201 @@
...
+- **Évitez les sliders 1/2 avec des reverses multiples.** Les nouveaux joueurs sont incapables de lire les reverses supplémentaires car ils sont visibles pendant un temps très court.
+- **Évitez les sections où il n'y a que des sliders.** Viser et suivre un grand nombre de sliders à la suite peut être fatigant pour les nouveaux joueurs. Dans ce cas, il convient d'utiliser des cercles et des moments de repos sans objets à cliquer ou à suivre.
+- **Les [stacks](/wiki/Mapping_techniques/Stack) sont acceptables, mais évitez d'alterner entre les stacks 1/1 et 1/2 de manière répétée.** Cela peut poser des problèmes de compréhension aux nouveaux joueurs, puisque deux rythmes différents sont visuellement identiques.
+- **La manipulation fréquente du [slider velocity](/wiki/Hit_object/Slider_velocity) est déconseillée.** Le slider velocity ne doit être modifiée que pour les sections de musique avec des rythmes différents et ne doit pas être radicalement variable.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!/fr.md:112]:
```suggestion
- **La manipulation fréquente du [slider velocity](/wiki/Hit_object/Slider_velocity) est déconseillée.** Le slider velocity ne doit être modifié que pour les sections de musique avec des rythmes différents et ne doit pas être radicalement variable.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 9]
```diff
@@ -0,0 +1,201 @@
...
+- **Utilisez des formes de sliders simples et facilement compréhensibles.** Les nouveaux joueurs peuvent ne pas comprendre comment suivre des formes plus complexes.
+- **Il doit y avoir au moins 2 temps entre l'extrémité d'un spinner et l'objet suivant.** Ceci afin de garantir un temps suffisant pour cliquer sur un objet qui suit un spinner.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!/fr.md:114]:
```suggestion
- **Il doit y avoir au moins 2 temps entre l'extrémité d'un spinner et l'objet suivant.** Ceci permet de garantir un temps suffisant pour cliquer sur un objet qui suit un spinner.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 10]
```diff
@@ -0,0 +1,201 @@
...
+- **Évitez l'[overlapping](/wiki/Mapping_techniques/Overlap) des objets avec d'autres éléments des skins par défaut et spécifiques à la beatmap.** Il s'agit de tous les éléments qui font partie de l'interface et qui peuvent être skinnée.
+
+Si un niveau de difficulté Normal est requis et utilisé comme le niveau de difficulté le plus bas d'une beatmap, il doit également suivre ces directives :
+
+- **Évitez d'avoir plus de trois rythmes de 1/2 clics actifs à la suite.**
+- **Incluez un écart de rythme de 1/1 ou plus pour chaque deux mesures de jeu.** L'indulgence est autorisée pour les sections de beatmaps comportant des sliders extrêmement longs, tels que des sliders de plus de 2 mesures.
+- **Évitez plus de quatre tranches de 1/2 consécutives.** Cela inclut les sliders d'inversion 1/2.
+- **Les objets composant les 1/2 rythmes cliqués activement doivent se [chevaucher](/wiki/Mapping_techniques/Overlap) les uns les autres sur le terrain de jeu.** Les rythmes 1/2 joués passivement qui ne se chevauchent pas, comme un sliderend suivi d'un cercle, sont acceptables.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!/fr.md:126]:
```suggestion
- **Les objets composant les 1/2 rythmes cliqués activement doivent se [chevaucher](/wiki/Mapping_techniques/Overlap) les uns les autres sur le playfield.** Les rythmes 1/2 joués passivement qui ne se chevauchent pas, comme un sliderend suivi d'un cercle, sont acceptables.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 11]
```diff
@@ -0,0 +1,201 @@
...
+- **N'utilisez pas de [flèches de slider qui ne suivent pas le chemin visible du slider](/wiki/Ranking_Criteria/osu!/img/Unintuitive_slider_reverse_arrow.png).** Ceux-ci déforment la direction d'un slider.
+- **Chaque slider doit avoir une trajectoire claire et visible à suivre du début à la fin.** Les sliders qui se chevauchent sans bordures de sliders claires et les sliders dont les sections individuelles sont illisibles ne peuvent pas être utilisés. La position finale d'un slider doit être claire en supposant que le joueur possède un skin qui rend les cercles de fin de slider totalement transparents. 
+  - Les sliders ambigus avec des cercles de suivi qui couvrent la totalité de la trajectoire du slider sont autorisés, à condition que les limites du slider soient simples.
+
+#### Directives
+
+- **Il doit y avoir au moins un temps entre l'extrémité d'un spinner et l'objet suivant.** Ceci afin de garantir un temps suffisant pour cliquer sur un objet qui suit un spinner.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!/fr.md:146]:
```suggestion
- **Il doit y avoir au moins un temps entre la fin d'un spinner et l'objet suivant.** Ceci permet de garantir un temps suffisant pour cliquer sur un objet qui suit un spinner.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 12]
```diff
@@ -0,0 +1,201 @@
...
+- **La densité des notes devrait consister principalement en des rythmes de 1/2, occasionnellement de 1/4, ou plus lents.**
+- **Évitez les [streams](/wiki/Beatmap/Pattern/Stream) composés de plus de 5 notes.** Des sliders d'inversion courts peuvent être utilisés en échange de ceux-ci lorsque la muisique le permet.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!/fr.md:148]:
```suggestion
- **Évitez les [streams](/wiki/Beatmap/Pattern/Stream) composés de plus de 5 notes.** Des reverse sliders courts peuvent être utilisés à la place de ceux-ci lorsque la musique le permet.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 13]
```diff
@@ -0,0 +1,201 @@
...
+  - Les sliders ambigus avec des cercles de suivi qui couvrent la totalité de la trajectoire du slider sont autorisés, à condition que les limites du slider soient simples.
+
+#### Directives
+
+- **Les objets séparés de 1/4 ou moins ne doivent pas se [chevaucher](/wiki/Mapping_techniques/Overlap) complètement,** surtout sur les difficultés plus simples de Insane.
+- **Évitez les [flèches de slider qui ne suivent pas le chemin visible du slider](/wiki/Ranking_Criteria/osu!/img/Unintuitive_slider_reverse_arrow.png).** Ceux-ci déforment la direction d'un slider.
+- **Évitez les [jumpes](/wiki/Beatmap/Pattern/Jump) et les [stream](/wiki/Beatmap/Pattern/Stream) dont les cercles ne se [chevauchent](/wiki/Mapping_techniques/Overlap) pas.** Il s'agit de techniques de beatmapping principalement réservées aux difficultés Expert.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!/fr.md:173]:
```suggestion
- **Évitez les [jumps](/wiki/Beatmap/Pattern/Jump) et les [streams](/wiki/Beatmap/Pattern/Stream) dont les cercles ne se [chevauchent](/wiki/Mapping_techniques/Overlap) pas.** Il s'agit de techniques de beatmapping principalement réservées aux difficultés Expert.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 14]
```diff
@@ -0,0 +1,201 @@
...
+- L'[Overall difficulty](/wiki/Beatmapping/Overall_difficulty) doit être entre 7 et 9.
+- L'[HP drain rate](/wiki/Beatmapping/HP_drain_rate) doit être entre 5 et 8.
+- Le [Circle size](/wiki/Beatmapping/Circle_size) doit être de 7 ou moins.
+
+### ![](/wiki/shared/diff/expert-o.png?20211215) Expert
+
+#### Règles
+
+- **Chaque slider doit avoir une trajectoire claire et visible à suivre du début à la fin.** Les sliders qui se chevauchent sans bordures de slider claires et les sliders dont les sections individuelles sont illisibles ne peuvent pas être utilisés. La position finale d'un slider doit être claire en supposant qu'un joueur possède un skin qui rend les cercles de fin de slider totalement transparents. 
+  - Si le cercle de suivi d'un slider couvre l'ensemble de la trajectoire du slider, le mouvement de la slider ball peut être ambigu.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!/fr.md:190]:
```suggestion
  - Si le follow circle d'un slider couvre l'ensemble de la trajectoire du slider, le mouvement de la slider ball peut être ambigu.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 15]
```diff
@@ -0,0 +1,201 @@
...
+
+#### Directives
+
+- **Évitez les [flèches de slider qui ne suivent pas le chemin visible du slider](/wiki/Ranking_Criteria/osu!/img/Unintuitive_slider_reverse_arrow.png).** Ceux-ci déforment la direction d'un slider.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!/fr.md:194]:
```suggestion
- **Évitez les [flèches de slider qui ne suivent pas le chemin visible du slider](/wiki/Ranking_Criteria/osu!/img/Unintuitive_slider_reverse_arrow.png).** Cela déforme la direction d'un slider.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 16]
```diff
@@ -0,0 +1,201 @@
...
+
+#### Règles
+
+- **Chaque slider doit avoir une trajectoire claire et visible à suivre du début à la fin.** Les sliders qui se chevauchent sans bordures de slider claires et les sliders dont les sections individuelles sont illisibles ne peuvent pas être utilisés. La position finale d'un slider doit être claire en supposant qu'un joueur possède un skin qui rend les cercles de fin de slider totalement transparents. 
+  - Les sliders ambigus avec des cercles de suivi qui couvrent la totalité de la trajectoire du slider sont autorisés, à condition que les limites du slider soient simples.
+
+#### Directives
+
+- **Les objets séparés de 1/4 ou moins ne doivent pas se [chevaucher](/wiki/Mapping_techniques/Overlap) complètement,** surtout sur les difficultés plus simples de Insane.
+- **Évitez les [flèches de slider qui ne suivent pas le chemin visible du slider](/wiki/Ranking_Criteria/osu!/img/Unintuitive_slider_reverse_arrow.png).** Ceux-ci déforment la direction d'un slider.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!/fr.md:172]:
```suggestion
- **Évitez les [flèches de slider qui ne suivent pas le chemin visible du slider](/wiki/Ranking_Criteria/osu!/img/Unintuitive_slider_reverse_arrow.png).** Cela déforme la direction d'un slider.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 17]
```diff
@@ -0,0 +1,201 @@
...
+- ![](/wiki/shared/diff/hard-o.png?20211215) Hard
+- ![](/wiki/shared/diff/insane-o.png?20211215) Insane
+- ![](/wiki/shared/diff/expert-o.png?20211215) Expert
+
+### ![](/wiki/shared/diff/easy-o.png?20211215) Easy
+
+#### Règles
+
+- **Les objets séparés de 1 temps ou moins ne doivent pas se [chevaucher](/wiki/Mapping_techniques/Overlap) complètement.**
+- **N'utilisez pas de [flèches d'inversion du slider qui ne suivent pas le chemin visible du slider](/wiki/Ranking_Criteria/osu!/img/Unintuitive_slider_reverse_arrow.png).** Ceux-ci déforment la direction d'un slider.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!/fr.md:66]:
```suggestion
- **N'utilisez pas de [flèches d'inversion du slider qui ne suivent pas le chemin visible du slider](/wiki/Ranking_Criteria/osu!/img/Unintuitive_slider_reverse_arrow.png).** Cela déforme la direction d'un slider.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 18]
```diff
@@ -0,0 +1,201 @@
...
+- L'[Approach rate](/wiki/Beatmapping/Approach_rate) doit être de 5 ou moins.
+- L'[Overall difficulty](/wiki/Beatmapping/Overall_difficulty) / L'[HP drain rate](/wiki/Beatmapping/HP_drain_rate) doit être compris entre 1 et 3.
+- Le [Circle size](/wiki/Beatmapping/Circle_size) doit être de 4 ou moins.
+
+### ![](/wiki/shared/diff/normal-o.png?20211215) Normal
+
+#### Règles
+
+- **Les objets séparés de 1 temps ou moins ne doivent pas se [chevaucher](/wiki/Mapping_techniques/Overlap) complètement.**
+- **N'utilisez pas de [flèches d'inversion de slider qui ne suivent pas le chemin du slider](/wiki/Ranking_Criteria/osu!/img/Unintuitive_slider_reverse_arrow.png).** Ceux-ci déforment la direction d'un slider.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!/fr.md:99]:
```suggestion
- **N'utilisez pas de [flèches d'inversion de slider qui ne suivent pas le chemin du slider](/wiki/Ranking_Criteria/osu!/img/Unintuitive_slider_reverse_arrow.png).** Cela déforme la direction d'un slider.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 19]
```diff
@@ -0,0 +1,201 @@
...
+- L'[Approach rate](/wiki/Beatmapping/Approach_rate) doit être entre 4 et 6.
+- L'[Overall difficulty](/wiki/Beatmapping/Overall_difficulty) / L'[HP drain rate](/wiki/Beatmapping/HP_drain_rate) doit être entre 3 et 5.
+- Le [Circle size](/wiki/Beatmapping/Circle_size) doit être de 5 ou moins.
+
+### ![](/wiki/shared/diff/hard-o.png?20211215) Hard
+
+#### Règles
+
+- **Les objets séparés de 1/2 ou moins ne doivent pas se [chevaucher](/wiki/Mapping_techniques/Overlap) complètement.** Les slider heads ou tails entièrement recouvertes par des slider tails sont exemptées, pour autant que leurs slider bodies soient visibles.
+- **N'utilisez pas de [flèches de slider qui ne suivent pas le chemin visible du slider](/wiki/Ranking_Criteria/osu!/img/Unintuitive_slider_reverse_arrow.png).** Ceux-ci déforment la direction d'un slider.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!/fr.md:140]:
```suggestion
- **N'utilisez pas de [flèches de slider qui ne suivent pas le chemin visible du slider](/wiki/Ranking_Criteria/osu!/img/Unintuitive_slider_reverse_arrow.png).** Cela déforme la direction d'un slider.
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 20]
```diff
@@ -0,0 +1,201 @@
...
+- **Évitez les sections où il n'y a que des sliders.** Viser et suivre un grand nombre de sliders à la suite peut être fatigant pour les nouveaux joueurs. Dans ce cas, il convient d'utiliser des cercles et des moments de repos sans objets à cliquer ou à suivre.
+- **Il est déconseillé de manipuler fréquemment le [slider velocity](/wiki/Hit_object/Slider_velocity).** Le slider velocity ne doit être modifié que pour les sections de musique avec des rythmes différents et ne doit pas être radicalement variable.
+- **Utilisez des formes de sliders simples et facilement compréhensibles.** Les nouveaux joueurs peuvent ne pas comprendre comment suivre des formes plus complexes.
+- **Il doit y avoir au moins 4 temps entre la fin d'un spinner et l'objet suivant.** Cela permet de garantir un temps suffisant pour cliquer sur un objet qui suit un spinner.
+- **Évitez les spinners de moins de 4 temps.** Les joueurs ont besoin de temps pour reconnaître qu'ils doivent commencer à tourner.
+- **Évitez de recouvrir entièrement les flèches des sliders avec des objets dans les 4 temps où la flèche est atteinte.** Cela donne aux joueurs suffisamment de temps pour reconnaître la flèche.
```
Comment (Biokax) [inline wiki/Ranking_Criteria/osu!/fr.md:83]:
```suggestion
- **Évitez de recouvrir entièrement les flèches inversées des sliders avec des objets dans les 4 temps où la flèche inversée est atteinte.** Cela donne aux joueurs suffisamment de temps pour reconnaître la flèche.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 21]
```diff
@@ -0,0 +1,201 @@
...
+- **Les [stacks](/wiki/Mapping_techniques/Stack) sont acceptables, mais évitez d'alterner entre les stacks 1/1 et 1/2 de manière répétée.** Cela peut poser des problèmes de compréhension aux nouveaux joueurs, puisque deux rythmes différents sont visuellement identiques.
+- **La manipulation fréquente du [slider velocity](/wiki/Hit_object/Slider_velocity) est déconseillée.** Le slider velocity ne doit être modifié que pour les sections de musique avec des rythmes différents et ne doit pas être radicalement variable.
+- **Utilisez des formes de sliders simples et facilement compréhensibles.** Les nouveaux joueurs peuvent ne pas comprendre comment suivre des formes plus complexes.
+- **Il doit y avoir au moins 2 temps entre l'extrémité d'un spinner et l'objet suivant.** Ceci permet de garantir un temps suffisant pour cliquer sur un objet qui suit un spinner.
+- **Évitez les spinners de moins de 3 temps.** Les joueurs ont besoin de temps pour reconnaître qu'ils doivent commencer à tourner.
+- **Évitez de recouvrir entièrement les flèches des sliders avec des objets dans les 3 temps où la flèche est atteinte.** Cela donne aux joueurs suffisamment de temps pour reconnaître la flèche.
```
Comment (Biokax) [inline wiki/Ranking_Criteria/osu!/fr.md:116]:
```suggestion
- **Évitez de recouvrir entièrement les flèches inversées des sliders avec des objets dans les 3 temps où la flèche inversée est atteinte.** Cela donne aux joueurs suffisamment de temps pour reconnaître la flèche.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 22]
```diff
@@ -0,0 +1,201 @@
...
+- **Évitez les [streams](/wiki/Beatmap/Pattern/Stream) composés de plus de 5 notes.** Des reverse sliders courts peuvent être utilisés à la place de ceux-ci lorsque la musique le permet.
+- **Évitez un espacement visuellement similaire pour des rythmes différents.** Les variations d'espacement par le biais de [jumps](/wiki/Beatmap/Pattern/Jump) sur les points d'emphase sont encouragées, mais seulement si ces jumps sont reconnaissables et différents des autres écarts rythmiques.
+- **Les slider tick hitsounds sont déconseillés.** Si vous souhaitez les utiliser, veillez à ce que leur volume soit équilibré (c'est-à-dire nettement plus faible que celui des hitsounds ordinaires). Un tick très fort du slider, surtout s'il n'est utilisé qu'une ou deux fois, peut être extrêmement déstabilisant.
+- **Évitez les spinners de moins de 2 temps.** Les joueurs ont besoin de temps pour reconnaître qu'ils doivent commencer à tourner.
+- **Évitez de recouvrir entièrement les flèches des sliders avec des objets dans les 2 temps où la flèche est atteinte.** Cela donne aux joueurs suffisamment de temps pour reconnaître la flèche.
```
Comment (Biokax) [inline wiki/Ranking_Criteria/osu!/fr.md:152]:
```suggestion
- **Évitez de recouvrir entièrement les flèches inversées des sliders avec des objets dans les 2 temps où la flèche inversée est atteinte.** Cela donne aux joueurs suffisamment de temps pour reconnaître la flèche.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 23]
```diff
@@ -0,0 +1,201 @@
...
+### ![](/wiki/shared/diff/insane-o.png?20211215) Insane
+
+#### Règles
+
+- **Chaque slider doit avoir une trajectoire claire et visible à suivre du début à la fin.** Les sliders qui se chevauchent sans bordures de slider claires et les sliders dont les sections individuelles sont illisibles ne peuvent pas être utilisés. La position finale d'un slider doit être claire en supposant qu'un joueur possède un skin qui rend les cercles de fin de slider totalement transparents. 
+  - Les sliders ambigus avec des cercles de suivi qui couvrent la totalité de la trajectoire du slider sont autorisés, à condition que les limites du slider soient simples.
+
+#### Directives
+
+- **Les objets séparés de 1/4 ou moins ne doivent pas se [chevaucher](/wiki/Mapping_techniques/Overlap) complètement,** surtout sur les difficultés plus simples de Insane.
```
Comment (Biokax) [inline wiki/Ranking_Criteria/osu!/fr.md:171]:
```suggestion
- **Les objets séparés de 1/4 ou moins ne doivent pas se [chevaucher](/wiki/Mapping_techniques/Overlap) complètement,** surtout sur les difficultés Insane plus simples.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 24]
```diff
@@ -0,0 +1,201 @@
...
+- **Évitez les [flèches de slider qui ne suivent pas le chemin visible du slider](/wiki/Ranking_Criteria/osu!/img/Unintuitive_slider_reverse_arrow.png).** Cela déforme la direction d'un slider.
```
Comment (Biokax) [inline wiki/Ranking_Criteria/osu!/fr.md:172]:
```suggestion
- **Évitez les [flèches inversées de slider qui ne suivent pas le chemin visible du slider](/wiki/Ranking_Criteria/osu!/img/Unintuitive_slider_reverse_arrow.png).** Cela déforme la direction d'un slider.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 25]
```diff
@@ -0,0 +1,201 @@
...
+- **Évitez les [jumps](/wiki/Beatmap/Pattern/Jump) et les [streams](/wiki/Beatmap/Pattern/Stream) dont les cercles ne se [chevauchent](/wiki/Mapping_techniques/Overlap) pas.** Il s'agit de techniques de beatmapping principalement réservées aux difficultés Expert.
+- **Les slider tick hitsounds sont déconseillés.** Si vous souhaitez les utiliser, veillez à ce que leur volume soit équilibré (c'est-à-dire nettement plus faible que celui des hitsounds ordinaires). Un tick très fort du slider, surtout s'il n'est utilisé qu'une ou deux fois, peut être extrêmement déstabilisant.
+- **Évitez de recouvrir entièrement les flèches des sliders avec des cercles, des sliders heads ou des sliders tails dans un délai de 1/2 avant que la flèche ne soit atteinte.** Cela donne aux joueurs suffisamment de temps pour reconnaître la flèche.
```
Comment (Biokax) [inline wiki/Ranking_Criteria/osu!/fr.md:175]:
```suggestion
- **Évitez de recouvrir entièrement les flèches inversées des sliders avec des cercles, des sliders heads ou des sliders tails dans un délai de 1/2 avant que la flèche inversée ne soit atteinte.** Cela donne aux joueurs suffisamment de temps pour reconnaître la flèche.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 26]
```diff
@@ -0,0 +1,201 @@
...
+### ![](/wiki/shared/diff/expert-o.png?20211215) Expert
+
+#### Règles
+
+- **Chaque slider doit avoir une trajectoire claire et visible à suivre du début à la fin.** Les sliders qui se chevauchent sans bordures de slider claires et les sliders dont les sections individuelles sont illisibles ne peuvent pas être utilisés. La position finale d'un slider doit être claire en supposant qu'un joueur possède un skin qui rend les cercles de fin de slider totalement transparents. 
+  - Si le follow circle d'un slider couvre l'ensemble de la trajectoire du slider, le mouvement de la slider ball peut être ambigu.
+
+#### Directives
+
+- **Évitez les [flèches de slider qui ne suivent pas le chemin visible du slider](/wiki/Ranking_Criteria/osu!/img/Unintuitive_slider_reverse_arrow.png).** Cela déforme la direction d'un slider.
```
Comment (Biokax) [inline wiki/Ranking_Criteria/osu!/fr.md:194]:
```suggestion
- **Évitez les [flèches inversées de slider qui ne suivent pas le chemin visible du slider](/wiki/Ranking_Criteria/osu!/img/Unintuitive_slider_reverse_arrow.png).** Cela déforme la direction d'un slider.
```

File: wiki/Ranking_Criteria/osu!/fr.md [hunk 27]
```diff
@@ -0,0 +1,201 @@
...
+- L'[Approach rate](/wiki/Beatmapping/Approach_rate) doit être entre 4 et 6.
+- L'[Overall difficulty](/wiki/Beatmapping/Overall_difficulty) / L'[HP drain rate](/wiki/Beatmapping/HP_drain_rate) doit être entre 3 et 5.
+- Le [Circle size](/wiki/Beatmapping/Circle_size) doit être de 5 ou moins.
+
+### ![](/wiki/shared/diff/hard-o.png?20211215) Hard
+
+#### Règles
+
+- **Les objets séparés de 1/2 ou moins ne doivent pas se [chevaucher](/wiki/Mapping_techniques/Overlap) complètement.** Les slider heads ou tails entièrement recouvertes par des slider tails sont exemptées, pour autant que leurs slider bodies soient visibles.
+- **N'utilisez pas de [flèches de slider qui ne suivent pas le chemin visible du slider](/wiki/Ranking_Criteria/osu!/img/Unintuitive_slider_reverse_arrow.png).** Cela déforme la direction d'un slider.
```
Comment (Biokax) [inline wiki/Ranking_Criteria/osu!/fr.md:140]:
```suggestion
- **N'utilisez pas de [flèches inversées de slider qui ne suivent pas le chemin visible du slider](/wiki/Ranking_Criteria/osu!/img/Unintuitive_slider_reverse_arrow.png).** Cela déforme la direction d'un slider.
```

**Review by Biokax — CHANGES_REQUESTED** [review]
reverse arrow -> flêche inversée

**Review by Biokax — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**